server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/pageobjects/WebhooksPage.java

```diff
@@ -42,6 +47,37 @@ public WebhooksPage countWebhooks(Integer number) {
     return this;
   }
 
+  public WebhooksPage hasNoWebhooks() {
+    $(".boxed-group").shouldHave(text("No webhook defined"));
+    return this;
+  }
+
+  public WebhooksPage createWebhook(String name, String url) {
+    $(".js-webhook-create").shouldBe(visible).shouldBe(enabled).shouldNotHave(cssClass("disabled")).click();
+    $(".modal-head").shouldBe(visible).shouldHave(text("Create Webhook"));
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Define a constant instead of duplicating this literal ".modal-head" 5 times. ([squid:S1192](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1192))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2FMMF-647%2FSONAR-10347_deliveries&issues=AWGK-3onuvYJzwp2ZHeN&open=AWGK-3onuvYJzwp2ZHeN)

[ISSUE_KEY]: AWGK-3onuvYJzwp2ZHeN